cmd/kubeadm/.import-restrictions

```diff
@@ -3,3 +3,4 @@ rules:
   - selectorRegexp: k8s[.]io/kubernetes
     allowedPrefixes:
       - k8s.io/kubernetes/cmd/kubeadm
+      - k8s.io/kubernetes/pkg/probe
```

Comment:
kubeadm must not import kubernetes/pkg

Reply:
Why kubeadm is limited from importing kubernetes/pkg

Reply:
Please see https://github.com/kubernetes/kubeadm/issues/1600

Reply:
kubernetes/pkg contains tools that must not be imported by external projects.
at the same time, we want to move kubeadm outside of kubernetes/kubernetes one day.